patches/chrome-browser-android-chrome_feature_list.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/android/chrome_feature_list.cc b/chrome/browser/android/chrome_feature_list.cc
+index 5a292bf4825f48e20513bcf6b313e080b0db5a9a..4e696dd969003256dae6ad2aa51e34b3f5f3d7f4 100644
+--- a/chrome/browser/android/chrome_feature_list.cc
++++ b/chrome/browser/android/chrome_feature_list.cc
+@@ -220,6 +220,7 @@ const base::Feature* kFeaturesExposedToJava[] = {
+ };
+ 
+ const base::Feature* FindFeatureExposedToJava(const std::string& feature_name) {
++  BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA
```

Comment:
This function is used in several places in this very same file, so I think I can't substitute this directly through chromium_src override.
I am not sure if there is a solution that we can do this without any patching, but this one line patch is the best I could think of at the moment, and this is extensible when we need to expose new feature flags to android.